Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -2751,6 +2740,13 @@ public IEnumerable<Symbol> LookupSymbols(string lookupName, SecurityType securit
             }
             else if (securityType == SecurityType.Future)
             {
+                string market;
+                if (_symbolPropertiesDatabase.TryGetMarket(contract.Symbol, securityType, out market))
+                {
+                    var symbolProperties = _symbolPropertiesDatabase.GetSymbolProperties(market, contract.Symbol, securityType, Currencies.USD);
+                    contract.Multiplier = Convert.ToInt32(symbolProperties.ContractMultiplier).ToStringInvariant();
```

Comment:
Curious why convert it to `int` instead of directly going from decimal to string?
The same comment about the `contract.Symbol` usages, maybe here we'll have to call the symbol mapper?

Reply:
Seems all multipliers are integers and IB didn't like the string "5000.0" 😄 

You're right about the `contract.Symbol` usage, needs to be the LEAN ticker 👍